### DISCUSSION TITLE
Is there any  way to give default config at organization level (each team can inherit from this for team repositories) ?

### DISCUSSION THREAD

**Discussion by anuj147256** [zricethezav/gitleaks#1020] [Q&A] [answered]
I have a use case where we want to have org wise default config (toml) and each team should inherit from this config.
I found this section to inherit from default gitleaks config.
```
[extend]
# useDefault will extend the base configuration with the default gitleaks config:
# https://github.com/zricethezav/gitleaks/blob/master/config/gitleaks.toml
useDefault = true
```
This allows me to import default config, but what I want is to have org level config say --> org-default.toml
org-default.toml extends the default gitleaks config
and then individual teams can extend from this "org-default.toml" to create team specific config.

This helps in case if there is a common secret across all repositories then it has to be added at one place i.e. org-default.toml

**Comment by weineran** [comment, marked as answer]
Yes, this is possible! See this blog post: https://blog.gitleaks.io/stop-leaking-secrets-configuration-2-3-aeed293b1fbf
In particular, the section "Extending a Central Config"

**Reply by anuj147256** [reply]
Thanks a ton , it worked. Better to document in main wiki.
